vercel/turbo
# chore: upgrade pnpm to version that support node 20

### Description

Before `pnpm@8.3.1` wasn't compatible with Node 20 (https://github.com/pnpm/pnpm/issues/6424)
Fixes #5587 #5592

I chose `8.6.10` as it will also include the fix for https://github.com/pnpm/pnpm/issues/6824 which affects `turbo prune`

### Testing Instructions

Did a quick spot check that the basic example works with the latest pnpm version and `pnpm install` will now work if using Node 20.
